ingestors/npm.go

```diff
@@ -43,9 +44,9 @@ type NPMChangeDoc struct {
 func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
 	since, err := ingestor.GetLatestSequence()
 	if err != nil {
-		log.Fatal(err)
+		log.WithFields(log.Fields{"ingestor": "npm"}).Fatal(err)
 	}
-	log.Printf("Depper Ingestor platform=npm sequence=%s", since)
+	log.WithFields(log.Fields{"platform": "npm", "sequence": since}).Info("Depper ingest")
```

Comment:
@jsonperl @mellisio what's your general take on doing this instead of printf-style interpolation?

Reply:
Love it, what's the output look like in the log?

Reply:
`INFO[2021-01-14T11:36:13-05:00] Depper ingest                                 platform=npm sequence=11206830`

Not a huge fan of that big padding in the middle but it's not the end of the world.

Reply:
Yea what's that padding all about?

Reply:
Can't figure out where that's set ATTM, but maybe we can revisit it later if it bugs us?